integration-tests/src/test/resources/OperatorIT.properties

```diff
@@ -16,6 +16,7 @@ oracledbImageTag = 12.2.0.1-slim
 oracledbImageName = container-registry.oracle.com/database/enterprise
 OCR_SERVER=container-registry.oracle.com
 DOMAIN_API_VERSION = weblogic.oracle/v7
+CRD_VERSION = v7
```

Comment:
can you get the version from develop crd instead of adding a property here, we will have to change this value for every release.

Reply:
Unless I start the domain in develop branch, is it possible to get the domain CRD offline.

Reply:
I don't understand the question, Pani.  Vanaja, I'm okay with having a separate test value here so that the test code doesn't depend on symbols from the production code.

Reply:
Ryan, still the value here is whatever is used in production code right.

Reply:
Yes, that's right.  They would need to be kept in sync.  Here's another option.  We make another module in maven with shared symbols like this that the operator runtime and integration tests could share.

Reply:
That's a good idea.

Reply:
Okay, I can make that happen but not in the PR.

Reply:
Ok. should we leave it hard coded for now or get the value from develop crd?

Reply:
Let's leave it hardcoded now please because I would strongly like to not have integration-test depend on the operator module.

Reply:
Will just starting the operator in develop branch create crd domains.weblogic.oracle?

Reply:
We will keep the value hard-coded till we get a shared module for crd version on the github repository